src/main/java/Point.java

```diff
@@ -0,0 +1,17 @@
+public class Point {
+    public double x;
+    public double y;
```

Comment:
Should be final if we want to access the members directly. If they are final the members should be capitalised according to [5.2.4 Constant names](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names).

An alternative way would be to encapsulate the members (i.e. make them private) and then expose them through getters and setters. This might be overkill for this simple class though.